### PR TITLE
Feature/new account ux

### PR DESCRIPTION
**Pull request by phys3**

### CODE REVIEW

File: src/components/shared/TwoButtonModal.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            </button>
+            <button
+              type="button"
+              className="Confirm"
+              data-dismiss="modal"
+              aria-label="Confirm"
+              onClick={handleConfirm}
+            >Continue
+            </button>
+</div>
```
Comment (TovarishFin) [inline src/components/shared/TwoButtonModal.js]: weird indent here...

File: src/components/shared/TwoButtonModal.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './TwoButtonModal.scss';
+
+const Modal = ({ isShowing, hide, title, text, handleConfirm }) =>
+  isShowing
+    ? 
+        <>
```
Comment (TovarishFin) [inline src/components/shared/TwoButtonModal.js]: why is this needed if you are already using a single tag surrounding everything else?

File: src/components/shared/TwoButtonModal.js (hunk 1 above)
Comment (phys3) [reply src/components/shared/TwoButtonModal.js]: believe it or not i never worked without prettier before

File: src/components/shared/TwoButtonModal.js (hunk 2 above)
Comment (phys3) [reply src/components/shared/TwoButtonModal.js]: no need will remove tnx
